cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -323,16 +356,22 @@ def fn(v):
         df = df.withColumn(col, lookup(mapping)(df[col]))
     return df
 
-
+# %%
+# Data Preparation
+# ================
+#
```

Comment:
I think we can remove this heading because the underlying function comes under data pre-processing only.